cabal-testsuite/PackageTests/ExtraProgPath/setup.test.hs

```diff
@@ -0,0 +1,7 @@
+import Test.Cabal.Prelude
+
+-- Test that extra-prog-path overrides the path for pkg-config
+main = cabalTest $ do
+  skipIfWindows
```

Comment:
i would add a comment mentioning why is is skipped in windows

Reply:
done